app/models/loader/create_policy.rb

```diff
@@ -17,6 +17,8 @@ def call
       @loader.delete_shadowed_and_duplicate_rows
 
       @loader.store_policy_in_db
+
+      @loader.db.disconnect
```

Comment:
Generally when you see nested member access like this (`x.y.z`), it's a code smell that we're breaking [Demeter's law](https://en.wikipedia.org/wiki/Law_of_Demeter).

In this case, what we should do is add a new method to the loader that describes in our problem domain what we're doing (e.g. `@loader.release_db_connection` or `@loader.recycle_db_connection`).

**Importantly,** that method needs an extensive header comment explaining why this manual disconnect is necessary with relevant links to the bug in postgres, the wiki explaining the cache, etc. Otherwise the context is going to be lost to history.

I would also suggest adding that same detail to the commit description so that this information is really easy to uncover in the future.